tikv/get-started/install-tikv-using-binary.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Install TiKV Using Binary Files
+category: overview
+---
+
+# Install TiKV Using Binary Files
+
+This guide describes how to deploy a TiKV cluster using binary files.
+
+- To quickly understand and try TiKV, see [Deploy the TiKV cluster on a single machine](#deploy-the-tikv-cluster-on-a-single-machine).
+- To try TiKV out and explore the features, see [Deploy the TiKV cluster on multiple nodes for test](#deploy-the-tikv-cluster-on-multiple-nodes-for-test).
+
+## Deploy the TiKV cluster on a single machine
+
+This section describes how to deploy TiKV on a single machine installed with the Linux system. Take the following steps:
+
+1. Download the official binary package.
+
+    ```bash
+    # Download the package.
+    wget https://download.pingcap.org/tidb-latest-linux-amd64.tar.gz
+    wget http://download.pingcap.org/tidb-latest-linux-amd64.sha256
+
+    # Check the file integrity. If the result is OK, the file is correct.
+    sha256sum -c tidb-latest-linux-amd64.sha256
+
+    # Extract the package.
+    tar -xzf tidb-latest-linux-amd64.tar.gz
+    cd tidb-latest-linux-amd64
+    ```
+
+2. Start PD.
+
+    ```bash
+    ./bin/pd-server --name=pd1 \
+                    --data-dir=pd1 \
+                    --client-urls="http://127.0.0.1:2379" \
+                    --peer-urls="http://127.0.0.1:2380" \
+                    --initial-cluster="pd1=http://127.0.0.1:2380" \
+                    --log-file=pd1.log
+    ```
+
+3. Start TiKV.
+
+    To start the 3 TiKV instances, open a new terminal tab or window, come to the `tidb-latest-linux-amd64` directory, and start the instances using the following command:
+
+    ```bash
+    ./bin/tikv-server --pd-endpoints="127.0.0.1:2379" \
+                    --addr="127.0.0.1:20160" \
+                    --data-dir=tikv1 \
+                    --log-file=tikv1.log
+
+    ./bin/tikv-server --pd-endpoints="127.0.0.1:2379" \
+                    --addr="127.0.0.1:20161" \
+                    --data-dir=tikv2 \
+                    --log-file=tikv2.log
+
+    ./bin/tikv-server --pd-endpoints="127.0.0.1:2379" \
+                    --addr="127.0.0.1:20162" \
+                    --data-dir=tikv3 \
+                    --log-file=tikv3.log
+    ```
+
+You can use the [pd-ctl](https://github.com/pingcap/pd/tree/master/pdctl) tool to verify whether PD and TiKV are successfully deployed:
+
+```
+./bin/pd-ctl store -d -u http://127.0.0.1:2379
```

Comment:
seem we need to say that if we find that all the state of TiKV is "up", we can think the deployment is successful.

Reply:
Added. Thanks!